diff -urN epiphany-extensions-1.5.8.orig/po/LINGUAS epiphany-extensions-1.5.8/po/LINGUAS
--- epiphany-extensions-1.5.8.orig/po/LINGUAS	2005-02-28 03:00:19.263782224 +0100
+++ epiphany-extensions-1.5.8/po/LINGUAS	2005-02-28 03:00:42.415262664 +0100
@@ -1,3 +1,3 @@
 # please keep this list sorted alphabetically
 #
-bg ca cs da de el en_CA en_GB es fr hr hu it ja ko lt nl no pa pl pt pt_BR ru sk sq sr sr@Latn sv uk wa zh_CN
+bg ca cs da de el en_CA en_GB es fr hr hu it ja ko lt nb nl pa pl pt pt_BR ru sk sq sr sr@Latn sv uk wa zh_CN
