diff -urN epiphany-extensions-1.7.5.orig/po/LINGUAS epiphany-extensions-1.7.5/po/LINGUAS
--- epiphany-extensions-1.7.5.orig/po/LINGUAS	2005-08-28 19:21:04.768236424 +0200
+++ epiphany-extensions-1.7.5/po/LINGUAS	2005-08-28 19:21:40.660779928 +0200
@@ -1,3 +1,3 @@
 # please keep this list sorted alphabetically
 #
-bg ca cs da de el en_CA en_GB es fi fr gl hr hu it ja ko lt ne nl no pa pl pt pt_BR ru rw sk sq sr sr@Latn sv uk wa zh_CN zh_TW
+bg ca cs da de el en_CA en_GB es fi fr gl hr hu it ja ko lt nb ne nl pa pl pt pt_BR ru rw sk sq sr sr@Latn sv uk wa zh_CN zh_TW
