diff -urN aa/epiphany-extensions-0.7.1/configure.ac epiphany-extensions-0.7.1/configure.ac
--- aa/epiphany-extensions-0.7.1/configure.ac	2004-02-15 13:57:32.000000000 +0100
+++ epiphany-extensions-0.7.1/configure.ac	2004-02-15 19:43:08.987208304 +0100
@@ -55,7 +55,7 @@
 dnl Internationalisation
 dnl ********************************
 
-ALL_LINGUAS="cs da de en_GB es fr ja nl no pt sk sr sr@Latn sv"
+ALL_LINGUAS="cs da de en_GB es fr ja nb nl pt sk sr sr@Latn sv"
 
 GETTEXT_PACKAGE=epiphany-extensions-1.0
 AC_SUBST(GETTEXT_PACKAGE)
