diff -urN aa/epiphany-extensions-1.4.0/extensions/smart-bookmarks/mozilla/Makefile.am epiphany-extensions-1.4.0/extensions/smart-bookmarks/mozilla/Makefile.am
--- aa/epiphany-extensions-1.4.0/extensions/smart-bookmarks/mozilla/Makefile.am	2004-09-07 21:08:07.000000000 +0200
+++ epiphany-extensions-1.4.0/extensions/smart-bookmarks/mozilla/Makefile.am	2004-09-13 17:04:36.059462032 +0200
@@ -5,6 +5,7 @@
 	-I$(MOZILLA_INCLUDE_ROOT)/content		\
 	-I$(MOZILLA_INCLUDE_ROOT)/docshell		\
 	-I$(MOZILLA_INCLUDE_ROOT)/dom			\
+	-I$(MOZILLA_INCLUDE_ROOT)/webbrwsr		\
 	-I$(MOZILLA_INCLUDE_ROOT)/xpcom			\
 	-I$(MOZILLA_INCLUDE_ROOT)			\
 	$(EPIPHANY_DEPENDENCY_CFLAGS) 			\
