diff -urN epiphany-extensions-2.22.2/extensions/error-viewer/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/error-viewer/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/error-viewer/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/error-viewer/mozilla/Makefile.am	2008-07-08 13:26:21.000000000 +0200
@@ -19,6 +19,7 @@
 	jsconsole	\
 	necko		\
 	webbrwsr	\
+	xpcom		\
 	xpconnect
 
 liberrorviewermozilla_la_CPPFLAGS = \
diff -urN epiphany-extensions-2.22.2/extensions/greasemonkey/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/greasemonkey/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/greasemonkey/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/greasemonkey/mozilla/Makefile.am	2008-07-08 13:20:44.000000000 +0200
@@ -9,6 +9,7 @@
 	content		\
 	docshell	\
 	dom		\
+	gtkembedmoz	\
 	xpcom
 
 libgreasemonkeymozilla_la_CPPFLAGS = \
diff -urN epiphany-extensions-2.22.2/extensions/java-console/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/java-console/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/java-console/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/java-console/mozilla/Makefile.am	2008-07-08 13:21:50.000000000 +0200
@@ -6,6 +6,7 @@
 
 mozilla_include_subdirs = \
 	.		\
+	gtkembedmoz	\
 	java		\
 	oji		\
 	xpcom
diff -urN epiphany-extensions-2.22.2/extensions/livehttpheaders/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/livehttpheaders/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/livehttpheaders/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/livehttpheaders/mozilla/Makefile.am	2008-07-08 13:22:05.000000000 +0200
@@ -8,6 +8,7 @@
 
 mozilla_include_subdirs = \
 	.		\
+	gtkembedmoz	\
 	necko		\
 	xpcom
 
diff -urN epiphany-extensions-2.22.2/extensions/page-info/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/page-info/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/page-info/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/page-info/mozilla/Makefile.am	2008-07-08 13:30:33.000000000 +0200
@@ -18,7 +18,8 @@
 	pref		\
 	uconv		\
 	webbrwsr	\
-	widget
+	widget		\
+	xpcom
 
 libpageinfomozilla_la_CPPFLAGS = \
 	-I..						\
diff -urN epiphany-extensions-2.22.2/extensions/sample-mozilla/mozilla/Makefile.am epiphany-extensions-2.22.2.new/extensions/sample-mozilla/mozilla/Makefile.am
--- epiphany-extensions-2.22.2/extensions/sample-mozilla/mozilla/Makefile.am	2008-05-26 20:08:15.000000000 +0200
+++ epiphany-extensions-2.22.2.new/extensions/sample-mozilla/mozilla/Makefile.am	2008-07-08 13:22:40.000000000 +0200
@@ -9,6 +9,7 @@
 	content		\
 	docshell	\
 	dom		\
+	gtkembedmoz	\
 	xpcom
 
 libsamplemozillamozilla_la_CPPFLAGS = \
diff -urN epiphany-extensions-2.22.2/m4/gecko.m4 epiphany-extensions-2.22.2.new/m4/gecko.m4
--- epiphany-extensions-2.22.2/m4/gecko.m4	2008-03-26 22:59:02.000000000 +0100
+++ epiphany-extensions-2.22.2.new/m4/gecko.m4	2008-07-08 13:18:46.000000000 +0200
@@ -116,7 +116,7 @@
 	_GECKO_PREFIX="`$PKG_CONFIG --variable=prefix ${gecko_cv_gecko}-xpcom`"
 	_GECKO_NSPR=no # XXX asac: this is currently a blind guess and should be a AC test
 else
-	_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`/unstable"
+	_GECKO_INCLUDE_ROOT="`$PKG_CONFIG --variable=includedir ${gecko_cv_gecko}`"
 	_GECKO_CFLAGS="`$PKG_CONFIG --cflags ${gecko_cv_gecko}` `$PKG_CONFIG --cflags ${gecko_cv_gecko}-unstable`"
 	_GECKO_LIBDIR="`$PKG_CONFIG --variable=sdkdir ${gecko_cv_gecko}`/bin"
 	_GECKO_HOME=$with_gecko_home
