diff -urN epiphany-extensions-1.1.3.orig/configure.ac epiphany-extensions-1.1.3/configure.ac
--- epiphany-extensions-1.1.3.orig/configure.ac	2004-08-16 21:08:00.479820800 +0200
+++ epiphany-extensions-1.1.3/configure.ac	2004-08-16 21:08:14.410702984 +0200
@@ -60,7 +60,7 @@
 dnl Internationalisation
 dnl ********************************
 
-ALL_LINGUAS="bg ca cs da de el en_CA en_GB es fr hr hu it ja ko lt nl no pa pl pt pt_BR ru sk sq sr sr@Latn sv uk wa zh_CN"
+ALL_LINGUAS="bg ca cs da de el en_CA en_GB es fr hr hu it ja ko lt nb nl pa pl pt pt_BR ru sk sq sr sr@Latn sv uk wa zh_CN"
 
 GETTEXT_PACKAGE=epiphany-extensions-1.2
 AC_SUBST(GETTEXT_PACKAGE)
