diff -urN epiphany-extensions-1.7.5.orig/extensions/adblock/mozilla/Makefile.am epiphany-extensions-1.7.5/extensions/adblock/mozilla/Makefile.am
--- epiphany-extensions-1.7.5.orig/extensions/adblock/mozilla/Makefile.am	2005-08-28 20:49:56.203735032 +0200
+++ epiphany-extensions-1.7.5/extensions/adblock/mozilla/Makefile.am	2005-08-28 22:02:24.092755432 +0200
@@ -11,6 +11,8 @@
 	content		\
 	docshell	\
 	dom		\
+	necko		\
+	webbrwsr	\
 	xpcom
 
 libadblockmozilla_la_CPPFLAGS = \
diff -urN epiphany-extensions-1.7.5.orig/extensions/auto-scroller/mozilla/Makefile.am epiphany-extensions-1.7.5/extensions/auto-scroller/mozilla/Makefile.am
--- epiphany-extensions-1.7.5.orig/extensions/auto-scroller/mozilla/Makefile.am	2005-08-28 20:49:56.494690800 +0200
+++ epiphany-extensions-1.7.5/extensions/auto-scroller/mozilla/Makefile.am	2005-08-28 20:56:30.391809336 +0200
@@ -9,6 +9,7 @@
 	content		\
 	docshell	\
 	dom		\
+	webbrwsr	\
 	xpcom
 
 libautoscrollermozilla_la_CPPFLAGS = \
diff -urN epiphany-extensions-1.7.5.orig/extensions/error-viewer/mozilla/Makefile.am epiphany-extensions-1.7.5/extensions/error-viewer/mozilla/Makefile.am
--- epiphany-extensions-1.7.5.orig/extensions/error-viewer/mozilla/Makefile.am	2005-08-28 20:49:53.852092536 +0200
+++ epiphany-extensions-1.7.5/extensions/error-viewer/mozilla/Makefile.am	2005-08-28 20:54:26.401658704 +0200
@@ -16,7 +16,8 @@
 	.		\
 	dom		\
 	jsconsole	\
-	necko
+	necko		\
+	webbrwsr
 
 liberrorviewermozilla_la_CPPFLAGS = \
 	-I..						\
diff -urN epiphany-extensions-1.7.5.orig/extensions/page-info/mozilla/Makefile.am epiphany-extensions-1.7.5/extensions/page-info/mozilla/Makefile.am
--- epiphany-extensions-1.7.5.orig/extensions/page-info/mozilla/Makefile.am	2005-08-28 20:49:53.625127040 +0200
+++ epiphany-extensions-1.7.5/extensions/page-info/mozilla/Makefile.am	2005-08-28 22:14:59.178964880 +0200
@@ -14,6 +14,7 @@
 	layout		\
 	nkcache		\
 	necko		\
+	pref		\
 	uconv		\
 	webbrwsr
 
diff -urN epiphany-extensions-1.7.5.orig/extensions/select-stylesheet/mozilla/Makefile.am epiphany-extensions-1.7.5/extensions/select-stylesheet/mozilla/Makefile.am
--- epiphany-extensions-1.7.5.orig/extensions/select-stylesheet/mozilla/Makefile.am	2005-08-28 20:49:55.301872136 +0200
+++ epiphany-extensions-1.7.5/extensions/select-stylesheet/mozilla/Makefile.am	2005-08-28 20:55:09.807060080 +0200
@@ -6,6 +6,7 @@
 
 mozilla_include_subdirs = \
 	.		\
+	dom		\
 	necko		\
 	nkcache		\
 	webbrwsr
