diff -urN aa/epiphany-extensions-1.5.7/extensions/error-viewer/mozilla/Makefile.am epiphany-extensions-1.5.7/extensions/error-viewer/mozilla/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/error-viewer/mozilla/Makefile.am	2004-12-08 16:33:52.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/error-viewer/mozilla/Makefile.am	2005-02-08 17:20:49.260647760 +0100
@@ -16,7 +16,8 @@
 	.		\
 	dom		\
 	jsconsole	\
-	necko
+	necko \
+	webbrwsr
 
 liberrorviewermozilla_la_CPPFLAGS = \
 	-I..						\
diff -urN aa/epiphany-extensions-1.5.7/extensions/page-info/mozilla/Makefile.am epiphany-extensions-1.5.7/extensions/page-info/mozilla/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/page-info/mozilla/Makefile.am	2004-12-08 16:33:52.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/page-info/mozilla/Makefile.am	2005-02-08 17:25:38.975604392 +0100
@@ -12,6 +12,7 @@
 	layout		\
 	nkcache		\
 	necko		\
+	pref		\
 	uconv		\
 	webbrwsr
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/select-stylesheet/mozilla/Makefile.am epiphany-extensions-1.5.7/extensions/select-stylesheet/mozilla/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/select-stylesheet/mozilla/Makefile.am	2004-12-08 16:33:53.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/select-stylesheet/mozilla/Makefile.am	2005-02-08 17:28:31.583364024 +0100
@@ -6,6 +6,7 @@
 
 mozilla_include_subdirs = \
 	.		\
+	dom			\
 	necko		\
 	nkcache		\
 	webbrwsr
