diff -urN aa/epiphany-extensions-1.5.7/extensions/actions/Makefile.am epiphany-extensions-1.5.7/extensions/actions/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/actions/Makefile.am	2005-02-01 15:17:45.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/actions/Makefile.am	2005-02-11 01:03:01.917191336 +0100
@@ -34,7 +34,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/adblock/Makefile.am epiphany-extensions-1.5.7/extensions/adblock/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/adblock/Makefile.am	2005-01-21 14:47:02.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/adblock/Makefile.am	2005-02-11 01:03:10.903825160 +0100
@@ -48,7 +48,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/bookmarks-tray/Makefile.am epiphany-extensions-1.5.7/extensions/bookmarks-tray/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/bookmarks-tray/Makefile.am	2005-01-11 23:43:20.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/bookmarks-tray/Makefile.am	2005-02-11 01:03:20.534361096 +0100
@@ -27,7 +27,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/certificates/Makefile.am epiphany-extensions-1.5.7/extensions/certificates/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/certificates/Makefile.am	2005-01-11 23:43:20.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/certificates/Makefile.am	2005-02-11 01:03:28.617132328 +0100
@@ -26,7 +26,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/dashboard/Makefile.am epiphany-extensions-1.5.7/extensions/dashboard/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/dashboard/Makefile.am	2005-01-11 23:43:20.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/dashboard/Makefile.am	2005-02-11 01:03:37.113840632 +0100
@@ -26,7 +26,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/error-viewer/Makefile.am epiphany-extensions-1.5.7/extensions/error-viewer/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/error-viewer/Makefile.am	2005-01-11 23:43:21.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/error-viewer/Makefile.am	2005-02-11 01:03:47.765221376 +0100
@@ -61,7 +61,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/extensions-manager-ui/Makefile.am epiphany-extensions-1.5.7/extensions/extensions-manager-ui/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/extensions-manager-ui/Makefile.am	2005-01-11 23:43:21.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/extensions-manager-ui/Makefile.am	2005-02-11 01:03:55.879987744 +0100
@@ -29,7 +29,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/find/Makefile.am epiphany-extensions-1.5.7/extensions/find/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/find/Makefile.am	2005-01-13 15:33:39.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/find/Makefile.am	2005-02-11 01:04:04.030748640 +0100
@@ -35,7 +35,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/gestures/Makefile.am epiphany-extensions-1.5.7/extensions/gestures/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/gestures/Makefile.am	2005-01-11 23:43:21.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/gestures/Makefile.am	2005-02-11 01:04:12.331486736 +0100
@@ -31,7 +31,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/net-monitor/Makefile.am epiphany-extensions-1.5.7/extensions/net-monitor/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/net-monitor/Makefile.am	2005-02-03 00:19:20.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/net-monitor/Makefile.am	2005-02-11 01:04:19.680369536 +0100
@@ -25,7 +25,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/page-info/Makefile.am epiphany-extensions-1.5.7/extensions/page-info/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/page-info/Makefile.am	2005-01-11 23:43:21.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/page-info/Makefile.am	2005-02-11 01:04:28.664003816 +0100
@@ -40,7 +40,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/sample/Makefile.am epiphany-extensions-1.5.7/extensions/sample/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/sample/Makefile.am	2005-01-11 23:43:22.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/sample/Makefile.am	2005-02-11 01:04:36.424823992 +0100
@@ -24,7 +24,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/sample-mozilla/Makefile.am epiphany-extensions-1.5.7/extensions/sample-mozilla/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/sample-mozilla/Makefile.am	2005-01-11 23:43:22.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/sample-mozilla/Makefile.am	2005-02-11 01:04:45.974372240 +0100
@@ -31,7 +31,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/select-stylesheet/Makefile.am epiphany-extensions-1.5.7/extensions/select-stylesheet/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/select-stylesheet/Makefile.am	2005-01-11 23:43:22.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/select-stylesheet/Makefile.am	2005-02-11 01:04:54.102136632 +0100
@@ -33,7 +33,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/sidebar/Makefile.am epiphany-extensions-1.5.7/extensions/sidebar/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/sidebar/Makefile.am	2005-01-11 23:43:23.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/sidebar/Makefile.am	2005-02-11 01:05:01.638990856 +0100
@@ -35,7 +35,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/smart-bookmarks/Makefile.am epiphany-extensions-1.5.7/extensions/smart-bookmarks/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/smart-bookmarks/Makefile.am	2005-01-14 00:12:39.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/smart-bookmarks/Makefile.am	2005-02-11 01:05:10.535638360 +0100
@@ -32,7 +32,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/tab-groups/Makefile.am epiphany-extensions-1.5.7/extensions/tab-groups/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/tab-groups/Makefile.am	2005-01-11 23:43:23.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/tab-groups/Makefile.am	2005-02-11 01:05:17.739543200 +0100
@@ -26,7 +26,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/tabsmenu/Makefile.am epiphany-extensions-1.5.7/extensions/tabsmenu/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/tabsmenu/Makefile.am	2005-01-11 23:43:23.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/tabsmenu/Makefile.am	2005-02-11 01:05:21.742934592 +0100
@@ -26,7 +26,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
diff -urN aa/epiphany-extensions-1.5.7/extensions/tab-states/Makefile.am epiphany-extensions-1.5.7/extensions/tab-states/Makefile.am
--- aa/epiphany-extensions-1.5.7/extensions/tab-states/Makefile.am	2005-01-13 15:33:40.000000000 +0100
+++ epiphany-extensions-1.5.7/extensions/tab-states/Makefile.am	2005-02-11 01:05:32.882241160 +0100
@@ -23,7 +23,7 @@
 %.xml.in: %.xml.in.in $(extension_LTLIBRARIES)
 	sed -e "s|%GETTEXT_PACKAGE%|$(GETTEXT_PACKAGE)|" \
 	    -e "s|%LOCALE_DIR%|$(datadir)/locale|" \
-	    -e "s|%LIBRARY%|`source $(extension_LTLIBRARIES) && echo $$dlname`|" \
+	    -e "s|%LIBRARY%|`. ./$(extension_LTLIBRARIES) && echo $$dlname`|" \
 	    -e "s|%EXTENSION_DIR%|$(extensiondir)|" \
 	$< > $@
 
